scripts/dts/extract/globals.py

```diff
@@ -351,38 +351,63 @@ def get_binding(node_address):
 def get_binding_compats():
     return bindings_compat
 
+def build_cell_array(prop_array):
+
```

Comment:
An empty line at the start of a function looks a bit weird to me.

Reply:
Fixed